crates/sui-framework/packages/sui-framework/sources/crypto/zklogin_verified_iss.move

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+module sui::zklogin_verified_iss {
+    use std::string::String;
+    use sui::object;
+    use sui::object::UID;
+    use sui::tx_context::TxContext;
+    use sui::tx_context;
+
+    /// Error if the proof consisting of the inputs provided to the verification function is invalid.
+    const EInvalidInput: u64 = 0;
+
+    /// Error if the proof consisting of the inputs provided to the verification function is invalid.
+    const EInvalidProof: u64 = 1;
+
+    /// Posession of a VerifiedIssuer proves that the user's address was created using zklogin and with the given issuer
+    /// (identity provider).
+    struct VerifiedIssuer has key {
+
+        /// The ID of this VerifiedIssuer
+        id: UID,
+
+        /// The address this VerifiedID is associated with
+        address: address,
+
+        /// The issuer
+        iss: String,
+    }
+
+    /// Returns the address associated with the given VerifiedIssuer
+    public fun get_address(verified_issuer: &VerifiedIssuer): address {
```

Comment:
We tend to follow Rust-style naming conventions, which would normally have accessors like this not have `get_` prefix, unless it is actually fetching something

```suggestion
    public fun address(verified_issuer: &VerifiedIssuer): address {
```

Reply:
This is not allowed because `address` is a protected keyword, so I've renamed the field to `owner`.